src/main/java/seedu/thanepark/ui/RideCard.java

```diff
@@ -54,12 +54,13 @@ public RideCard(Ride ride, int displayedIndex) throws AccessibilityException {
         statusString.setText(ride.getStatus().name());
         if (ride.getStatus().equals(Status.OPEN)) {
             statusString.setStyle("-fx-background-color: #27D858");
-        } else if (ride.getStatus().equals(Status.SHUTDOWN)
-            || ride.getStatus().equals(Status.MAINTENANCE)) {
+        } else if (ride.getStatus().equals(Status.SHUTDOWN)) {
             statusString.setStyle("-fx-background-color: #CC3045");
-        //error
+        } else if (ride.getStatus().equals(Status.MAINTENANCE)) {
+            statusString.setStyle("-fx-background-color: #cc6a26");
```

Comment:
Can we standardize? Lower case or upper case hex will do.

Reply:
Ok i'll keep it to uppercase hex